plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/OAuth2SecurityConfig.java

```diff
@@ -67,6 +69,22 @@ public OAuth2SecurityConfig setToken(String token)
         return this;
     }
 
+    public Optional<URI> getServerUri()
+    {
+        return Optional.ofNullable(serverUri);
+    }
+
+    @Config("iceberg.rest-catalog.oauth2.server-uri")
+    @ConfigDescription("The endpoint to retrieve access token from OAuth2 Server")
+    public OAuth2SecurityConfig setServerUri(String uri)
```

Comment:
The airlift which is used as a configuration framework accepts `URI` type. 
Please change to `URI` and rename to `serverUri`.